src/main/java/com/softserveinc/ita/pageobjects/admin/AddingNewQuestionPage.java

```diff
@@ -0,0 +1,71 @@
+package com.softserveinc.ita.pageobjects.admin;
+
+import com.softserveinc.ita.models.AnswerEntity;
+import io.qameta.allure.Step;
+
+import java.io.File;
+
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.lang.String.format;
+import static java.time.Duration.ofSeconds;
+
+public class AddingNewQuestionPage extends MainMenu<AddingNewQuestionPage> {
+
+    @Step("Adding New Question page: Filled in text of question")
+    public AddingNewQuestionPage fillTextOfQuestion(String textOfQuestion) {
+        var textField = $("textarea[formcontrolname='question_text']");
+        textField.clear();
```

Comment:
you could create seperate method with at MainMenu class

setValue(SelenideElelement element, String text) {
element.clear();
element.sendKeys(text);
}

Reply:
and replace such usages